.github/workflows/docs-build-test.yml

```
@@ -0,0 +1,37 @@
+name: Build Test for docs
+
+on:
+    push:
+      branches-ignore:
+        - 'master'
+      paths:
+        - 'docs/**'
+    pull_request:
+      branches:
+        - 'dev'
+      paths:
+        - 'docs/**'
+    workflow_dispatch:
+  
+jobs:
+  build-test:
+    timeout-minutes: 20
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v3
```

```suggestion
        uses: actions/checkout@v4
        with:
          show-progress: false
```
建议使用新版本，并且禁用过程显示（主仓库内容实在是太多了）